receiver/mongodbreceiver/ingegration_test.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build integration
```

Comment:
Please double check the name of this file.

Reply:
ok